auto-scale-aws.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+# Restarts AWS instance and changes the volume
+# Volume will be changed to t2.small if current season is "Off season"
+# The volume will be t2.medium if another season is active.
+
+
+# Using this function to remove double quotes from strings
+remove_double_quotes() {
+    echo $(sed -e 's/^"//' -e 's/"$//' <<<"$1")
+}
+
+off_season="Off season"
+instance_type_micro="t2.micro"
+instance_type_small="t2.small"
+instance_id="i-0ae22e94380789482"
+
+
+# Get instance volume
+instance_info=$(aws ec2 describe-instances --instance-ids  $instance_id --query "Reservations[*].Instances[*].{Status:State.Name,Instancetype:InstanceType}")
+instance_type=$(jq '.[0][0].Instancetype' <<< $instance_info)
+instance_type=$(remove_double_quotes "$instance_type")
+echo "InstanceType is $instance_type"
+
+
+# Fetch current_seanson. 
+response=$(curl http://diamonds.etimo.se/api/seasons/current)
+current_season=$(jq '.data.name'  <<< $response)
+current_season=$(remove_double_quotes "$current_season")
+
+if [ "$current_season" = "$off_season" ]; then
```

Comment:
Shouldnt it be double equality here? `==`

Reply:
Single equal sign works aswell. But changing to double to be more clear.